Vishnu_45/html/Ass6/assignment1.html

```diff
@@ -1,59 +1,46 @@
-<!DOCTYPE Html>
-  <head>
-    <title>
-      Page
-    </title>
-  </head>
-  <style>
-    h1 {
-    background-color: rgba(255, 255, 255, 0.925);
-  }
-  h2 {
-    background-color: rgba(255, 255, 255, 0.925);
-  }
-  </style>
-  <body bgcolor="Lightskyblue">
-
-    <form>
-      <header class="header">
-        <h1 align="right">MI-9-Intern HTML Assignment</h1>
-      </header>
-      <br>
-      <hr>
-      <header class="header">
-        <h2 align="right">Personal Details</h2>
-      </header>
-      <label> Firstname </label>
-      <input type="text" name="firstname" size="15" /> <br> <br>
-      <label> Lastname: </label>
-      <input type="text" name="lastname" size="15" /> <br> <br>
-      <label>
-        Gender :
-      </label>
-      <input type="radio" name="male" /> Male
-      <input type="radio" name="female" /> Female
-      <input type="radio" name="other" /> Other
-      <br>
-      <br>
-
-      <label>
-        Age :
-      </label>
-      <input type="radio" name="number" /> less than 18
-      <input type="radio" name="number" /> 18-25
-      <input type="radio" name="number" /> 26-40
-      <header class="header">
-        <h2 align="right">Academic Details</h2>
-      </header>
-      <label>
-        skill Set :
-      </label>
-      <input type="checkbox" name="HTML" /> HTML
-      <input type="checkbox" name="Java Script" /> Java Script 
-      <input type="checkbox" name="Java" /> Java
-      <input type="checkbox" name="Oracle" /> Oracle
-      <br>
-      <br>
-    </form>
-  </body>
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Page</title>
+    </head>
+    <style>
+        h1 {
+            background-color: rgba(255, 255, 255, 0.925);
+        }
+        h2 {
+            background-color: rgba(255, 255, 255, 0.925);
+        }
+    </style>
+    <body bgcolor="Lightskyblue">
+        <form>
+            <header class="header">
+                <h1 align="right">MI-9-Intern HTML Assignment</h1>
+            </header>
+            <br />
+            <hr />
+            <header class="header">
+                <h2 align="right">Personal Details</h2>
```

Comment:
Here  is no need to header tag, so remove it. & align is also out dated. so, remove from every where.